Modelica/Mechanics/MultiBody/Types/VectorQuantity.mo

```diff
@@ -6,4 +6,30 @@ type VectorQuantity = enumeration(
     Acceleration,
     AngularVelocity,
```

Comment:
Just noticed, that description of the alternatives might also be valuable. Will do later...